crates/next-core/src/next_config.rs

```diff
@@ -801,8 +801,8 @@ pub struct ExperimentalConfig {
     global_not_found: Option<bool>,
     /// Defaults to false in development mode, true in production mode.
     turbopack_remove_unused_exports: Option<bool>,
-    /// Devtool option for the segment explorer.
-    devtool_segment_explorer: Option<bool>,
+    /// Devtool option for new panel UI.
+    devtool_new_panel_ui: Option<bool>,
```

Comment:
```suggestion
    /// Devtool option for new panel UI.
     #[serde(rename = "devtoolNewPanelUI")]
    devtool_new_panel_ui: Option<bool>,
```
and then we can keep the public flag name.